redisearch.sh

```
@@ -0,0 +1,12 @@
+#!/bin/bash
+make -p redisearch
+cd redisearch
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)